src/library/scala/math/BigInt.scala

```diff
@@ -121,8 +123,8 @@ final class BigInt(val bigInteger: BigInteger)
     if (isValidLong) unifiedPrimitiveHashcode
     else bigInteger.##
 
-  /** Compares this BigInt with the specified value for equality.
-   */
+  /** Compares this BigInt with the specified value for equality. */
+  @nowarn("cat=other-non-cooperative-equals")
```

Comment:
can't actually do this yet, as it doesn't compile pre-bootstreap. will have to filter on message until re-STARR